src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/ProxyHeaderModule.Framework.cs

```diff
@@ -20,8 +20,8 @@ internal class ProxyHeaderModule : IHttpModule
     private const string ServerPort = "SERVER_PORT";
     private const string ForwardedProto = "x-forwarded-proto";
     private const string ForwardedHost = "x-forwarded-host";
-    private const string On = "ON";
-    private const string Off = "OFF";
+    private const string On = "on";
```

Comment:
The docs at https://docs.microsoft.com/en-us/iis/web-dev-reference/server-variables say `ON` and `OFF`. I didn't actually try that in a request (I'll do that soon), but I'm surprised if framework requires this to be lowercase given the docs have capitals.

Reply:
I just saw your link to the source of framework where it is indeed case sensitivt. Weird. I'm still going to do a sanity check on it (lesson learned: don't rely on docs for old technology)